### DISCUSSION TITLE
Unable to send emails using SMTP

### DISCUSSION THREAD

**Discussion by abishekrsrikaanth** [postalserver/postal#2663] [Help with using Postal]
I was able to get Postal setup perfectly and I am able to login to Postal Admin and setup everything. 
I am able to send test messages from within the Postal Admin, but when I try to connect to the Postal SMTP server from outside, it fails. Not sure what the issue is.

I went through the docs https://docs.postalserver.io/other/debugging to understand if I am missing something. Both `telnet` and `openssl` commands don't seem to work on my end. 

The telnet command just shows `Trying ipv6-address .....` and the same with the openssl command.

I have the both incoming and outgoing ports for 25 open. 

here is the output for `netstat`

```
netstat -lnt | grep 25
tcp        0      0 127.0.0.1:25            0.0.0.0:*               LISTEN
```

and the output for `iptables`

```
sudo iptables -nL | grep 25
[sudo] password for forge:
ACCEPT     tcp  --  0.0.0.0/0            0.0.0.0/0            tcp dpt:25
ACCEPT     udp  --  0.0.0.0/0            0.0.0.0/0            udp dpt:25
```

When i am tailing the postal logs I do see this error

```
postal-smtp-1      |
postal-smtp-1      | Errno::EADDRINUSE: Address already in use - bind(2) for "::" port 25
postal-smtp-1      |
postal-smtp-1      | /opt/postal/app/lib/postal/smtp_server/server.rb:51:in `initialize'
postal-smtp-1      | /opt/postal/app/lib/postal/smtp_server/server.rb:51:in `open'
postal-smtp-1      | /opt/postal/app/lib/postal/smtp_server/server.rb:51:in `listen'
postal-smtp-1      | /opt/postal/app/lib/postal/smtp_server/server.rb:285:in `run'
postal-smtp-1      | /opt/postal/app/lib/tasks/postal.rake:15:in `block (2 levels) in <top (required)>'
```

Can someone help understand what the issue could be.

**Comment by willpower232** [comment]
if you add `p` to your `netstat` command it will let you know which program is using the port, it is likely that your server OS installed postfix or exim by default so the port isn't available for docker/Postal to use
